__tests__/endpoints.test.js

```diff
@@ -218,3 +218,22 @@ describe("/api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("/api/users", () => {
+  test("GET 200 - Responds with a list of all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toHaveProperty("username");
```

Comment:
your typeof assertions are enough here as the property existence is implied from the values not being undefined